packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsSynchronizationStatus.tsx

```diff
@@ -3,17 +3,23 @@ import { Status } from '@/ui/display/status/components/Status';
 
 export type SettingsAccountsSynchronizationStatusProps = {
   syncStatus: string;
+  isSyncEnabled?: boolean | undefined;
```

Comment:
this should be isSyncedEnabled: boolean

CalendarChannel and MessageChannel should have a isSyncEnabled: boolean